topics/genome-annotation/tutorials/gene-centric/tutorial.md

```diff
@@ -0,0 +1,4048 @@
+
+It produces two primary outputs:
+
+> ### {% icon code-out %} Results
+>
+> 1. <kbd>Trees</kbd> - Phylogenetic trees for each input exon as [Newick](https://en.wikipedia.org/wiki/Newick_format) file
+> 1. <kbd>PlottingData</kbd> - A summary table of exon matches for each genome
+>
+{: .code-out}
+
+The overall logic of the workflow is as follows:
+
+1. Perform [__Diamond__](https://github.com/bbuchfink/diamond) search to identify matches between the gene of interest (<kbd>EXONS</kbd>) and ORF sets from each genomes (<kbd>DiamondDB</kbd>).
+2. Intersect information about the matches with BED file containing ORF coordinates (<kbd>ORF BED</kbd>). This allows us to know genomic position of the ORFs and their frames (1, 2, 3 or -1, -2, -3).
+3. Extract matching parts of the sequences and generated multiple alignments using [__MAFFT__](https://mafft.cbrc.jp/alignment/software/). This is done for each amino acid sequence in <kbd>EXONS</kbd>. Thus in the case of *XBP-1* there will be five sets of alignments. 
+4. Build phylogenetic tree for each alignment produced in the previous step using the [Neighbor Joining method](https://en.wikipedia.org/wiki/Neighbor_joining) - a simple and quick way to obtain a rough phylogeny.
+5. Use information from step 1 to compute genome coordinates of matches. This is done by combining the information about genomic positions of ORFs from the <kbd>ORF BED</kbd> file and local alignment information generated during step 1.
+
+## Step 7: Looking at the trees
+
+After running the workflow phylogenetic trees will be saved into a collection named `Join neighbors on ...`. This collection will also be labelled with tag <kbd>Trees</kbd>. To visualize the trees:
+
+> ### {% icon hands_on %} Visualize the trees
+>
+> - Expand the collection by clicking on it
+> - Click on any dataset
+> - Click the {% icon galaxy-barchart %} icon:
+>     ![Graph icon](../../images/gene-centric/display_tree.png)
+> - A selection of tree visualization tools will appear in the center pane of the interface
+> - Select `Phylogenetic Tre Visualization` and play with it:
+>     ![The tree](../../images/gene-centric/tree.png)
+{: .hands_on}
+
+## Step 8: Generating and interpreting the comparative genome graph
+
+Another workflow output will represent a single file summarizing genomic location of matches between each of the genomes in our dataset and amino acid translation of exons from the gene of interest. It will be called `Mapping report` and will have tag <kbd>PlottingData</kbd> associated with it. To plot the data contained in this file we will use external Jupyter notebook (note that Jupyter can be run directly from Galaxy, but to make this tutorial runnable on any Galaxy instance we will use an internal notebook server). 
+
+### Starting notebook
+
+The notebook can be accessed [from here](https://colab.research.google.com/drive/1smTpRejBb7c02LiIxMPNVDjOucLAsD81?usp=sharing). You do need to have a Google Account to be able to use the notebook.
```

Comment:
I'm not a fan either but this is easy and quick. This viz is critical for this tutorial. Currently IE do not work on main but they will be soon. Once they do I will change the tutorial. I really need this out soon as this is a part of our VGP effort.

Reply:
I'm reading that as a "no not open to splitting the notebook into a separate tutorial"

Then can you add the .ipynb file next to the tutorial at least? And then we can add a box for how one imports that into ITs since they work fine on the other servers, and not force everyone into using a google account authorized service.

Reply:
This is the ideal case for CYOA, you can have a lil box for "I'm using a galaxy with interactive tools" that has tip boxes for launching an interactive tool and loading the notebook from a URL, and then "I'm on main" as the other branch which points to colab.

Reply:
where should I put .ipynb file?

Reply:
> It just feels tough to accept "it doesn't work on main" as a reason to exclude a working path for the rest of the world.

Yes, I agree. I'll test this on EU and AU

Reply:
Please put it in this folder, next to the tutorial.md, that'll be fine for now!

CYOA = choose your own adventure. https://training.galaxyproject.org/training-material/news/2022/04/12/cyot.html

Reply:
I can write that bit, once you've added the IPYNB. no worries.